ui/features/mesheryEnvironment/components/KubernetesCluster/KubernetesClusterChip.js

```diff
@@ -4,15 +4,38 @@ import { Tooltip } from "@mui/material";
 import Image from "next/image";
 import { useTheme } from "@mui/system";
 import Chip from "@/components/Chip";
+import { useSnackbar } from 'notistack';
 
 /**
  * React component that takes in kubernetes clusters information among other things
  * as props and renders it in form of chip
  * @param {{cluster: import("@/features/mesheryEnvironment/mesheryEnvironmentSlice").KubernetesCluster}} props
  * @returns {import("react").ReactElement}
  */
+
 export const KuberenetesClusterChip = ({ cluster, handleClick }) => {
+
   const theme = useTheme();
+  const { enqueueSnackbar, closeSnackbar } = useSnackbar();
+  const handleKubernetesClick = () => {
```

Comment:
we should not write the function definition here, it should be passed as `handleClick`

Reply:
I didn't get it can you tell me a bit more on this.